services/harmony/env-defaults

```diff
@@ -135,6 +135,18 @@ MAX_DATA_OPERATION_CACHE_SIZE=512000000
 # Token Cache TTL in milliseconds
 TOKEN_CACHE_TTL=300000
 
+# Maximum size (in bytes) of the cache for CMR searches
+EDL_CACHE_SIZE=256000000
```

Comment:
I learned we can use _ in numbers to make them easier to read. It might be good to use them for all these numbers since it is hard to tell.
i.e. `256_000_000` - now I know this is 256MB.

Reply:
This only works in ts files, not here. I can put it in the docstring of the config variable.